packages/sit-onyx/src/components/OnyxBottomBar/OnyxBottomBar.vue

```diff
@@ -46,25 +48,33 @@ const { densityClass } = useDensity(props);
     position: sticky;
     bottom: 0;
     width: 100%;
-    margin-inline: var(--onyx-grid-margin-inline);
 
     background-color: var(--onyx-color-base-background-blank);
-    display: flex;
-    justify-content: space-between;
-    gap: var(--onyx-density-xs);
-    padding: var(--onyx-spacing-2xs) var(--onyx-grid-margin);
-    overflow-x: auto;
 
-    &--border {
-      border-top: var(--onyx-1px-in-rem) solid var(--onyx-color-component-border-neutral);
+    &--border::before {
+      content: " ";
+      background-color: var(--onyx-color-component-border-neutral);
+      height: var(--onyx-1px-in-rem);
+      width: 100%;
+      position: absolute;
     }
 
-    &__left,
-    &__right {
-      position: relative;
+    &__content {
       display: flex;
-      max-width: var(--onyx-grid-max-width, max-content);
+      justify-content: space-between;
       gap: var(--onyx-density-xs);
+      padding: var(--onyx-spacing-2xs) var(--onyx-grid-margin);
+      overflow-x: auto;
+      padding-inline: var(--onyx-grid-margin);
+      max-width: var(--onyx-grid-max-width);
+      margin-inline: var(--onyx-grid-margin-inline);
+
+      &--left,
+      &--right {
+        display: flex;
+        max-width: var(--onyx-grid-max-width, max-content);
```

Comment:
Why is this needed? The max width is already set on the parent so can we remove it here? Otherwise if both left and right slots are used, they would have two times the max grid width, right?